types/apminsight/index.d.ts

```diff
@@ -0,0 +1,96 @@
+// Type definitions for apminsight 3.0
+// Project: https://www.npmjs.com/package/apminsight
+// Definitions by: Site24x7 <https://github.com/site24x7>
```

Comment:
this cannot be an organization, the owner is a (technically) Github handle, with a person behind this, sorry!